test/manifest/encrypted.json

```diff
@@ -30,11 +33,21 @@
       "duration": 15,
       "key": {
         "method": "AES-128",
-        "uri": "https://priv.example.com/key.php?r=53"
+        "uri": "https://priv.example.com/key.php?r=53",
+        "iv": null
       },
       "uri": "http://media.example.com/fileSequence53-A.ts"
     },
     {
+      "duration": 14,
+      "key": {
+        "method": "AES-128",
+        "uri": "https://priv.example.com/key.php?r=54",
+        "iv": new Uint32Array([0, 0, 331, 3063767524])
```

Comment:
I just made it pass your test, if you want, you can remove the iv test completely, but then you have a missing test. The way you test is deep-comparing objects parsed from the m3u8 with the ones from the json file, i cannot nor should i rename the json file. I don't know of how you would have a byte/int array in a json file with it remaining JSON compliant either. You usually base64 encode binary data in JSON but then the deep-compare would fail.